README.md

```diff
@@ -87,6 +87,10 @@ Blank repository with a README:
 ```
 blank
 ```
+HTML boilerplate that comes CSS and JS ready:
```

Comment:
I don’t quite know why, but this just feels grammatically wrong. Maybe rewrite it as “HTML boilerplate complete with CSS and JS”?

Reply:
Oh, I forgot the word 'with' after 'that comes'... but yours sounds better!

Reply:
Yours sounds good, change it to that!